clustered/common/src/main/java/org/ehcache/clustered/common/internal/store/operations/codecs/OperationsCodec.java

```diff
@@ -37,6 +37,10 @@ public ByteBuffer encode(Operation<K, V> operation) {
     return operation.encode(keySerializer, valueSerializer);
   }
 
+  public static OperationCode getOperationCode(ByteBuffer buffer) {
+    return OperationCode.valueOf(buffer.duplicate().get());
+  }
+
```

Comment:
This assumes the buffer is at the right place. Then why not using `rewind` instead of duplicating? And you can probably call `getOperationCode` from `decode` now.